src/System.Windows.Forms/src/System/Windows/Forms/ListViewItem.ListViewItemAccessibleObject.cs

```diff
@@ -142,26 +145,195 @@ public override void DoDefaultAction()
+                   };
+            }
+
+            private int GetChildCountTileView()
+            {
+                // Data about the first ListViewSubItem is displayed in the ListViewItem.
+                // Therefore, it is not displayed in the ListViewSubItems list
+                if (_owningItem.SubItems.Count == 1)
+                {
+                    return 0;
+                }
+
+                return GetLastTileChildIndex();
+            }
+
+            private int GetChildIndex()
+            {
+                // Data about the first ListViewSubItem is displayed in the ListViewItem.
+                // Therefore, it is not displayed in the ListViewSubItems list
+                if (_owningItem.SubItems.Count == 1)
+                {
+                    return -1;
+                }
+
+                // Only ListViewSubItems with the corresponding columns are displayed in the ListView
+                int subItemCount = Math.Min(_owningListView.Columns.Count, _owningItem.SubItems.Count);
+
+                // The ListView can be of limited TileSize, so some of the ListViewSubItems can be hidden.
+                // sListViewSubItems that do not have enough space to display have an empty bounds
+                for (int i = 1; i < subItemCount; i++)
+                {
+                    if (GetSubItemBounds(i).IsEmpty)
+                    {
+                        return i - 1;
+                    }
+                }
+
+                return subItemCount - 1;
+            }
+
+            private int LastChildIndex
+                => _owningListView.View switch
+                {
+                    View.Details => _owningListView.Columns.Count - 1,
+                    View.LargeIcon => -1,
+                    View.List => -1,
+                    View.SmallIcon => -1,
+                    View.Tile => GetLastTileChildIndex(),
+                    _ => throw new NotImplementedException()
+                };
+
+            // This method returns an accessibility object for an existing ListViewSubItem, or creates a fake one
+            // if the ListViewSubItem does not exist. This is necessary for the "Details" view,
+            // when there is no ListViewSubItem, but the cell for it is displayed and the user can interact with it.
+            internal AccessibleObject? GetDetailsSubItemOrFake(int subItemIndex)
+            {
+                if (subItemIndex < _owningItem.SubItems.Count)
+                {
+                    _listViewSubItemAccessibleObjects.Remove(subItemIndex);
+
+                    return _owningItem.SubItems[subItemIndex].AccessibilityObject;
+                }
+
+                if (_listViewSubItemAccessibleObjects.ContainsKey(subItemIndex))
+                {
+                    return _listViewSubItemAccessibleObjects[subItemIndex];
+                }
+
+                ListViewSubItem.ListViewSubItemAccessibleObject fakeAccessbileObject = new(null, _owningItem);
```

Comment:
Please use the named args:
```suggestion
                ListViewSubItem.ListViewSubItemAccessibleObject fakeAccessbileObject = new(whats_my_name: null, _owningItem);
```